test_libs/pyspec/eth2spec/utils/ssz/ssz_typing.py

```diff
@@ -250,7 +255,7 @@ def __init__(self, *args: Iterable):
         # cannot check non-type objects, or parametrized types
         if isinstance(cls.elem_type, type) and not hasattr(cls.elem_type, '__args__'):
             for i, item in enumerate(self.items):
-                if not issubclass(type(item), cls.elem_type):
+                if not issubclass(cls.elem_type, type(item)):
```

Comment:
I think this should be `isintance(item, cls.elem_type)`. But is going to be more strict, less-newtype-workaround, and general with 1180 anyway.